src/plugins/prefs/autoconfig/easyTestSetup.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+TriggerPort=65432
+MountPoint="user/prefs"
+ConfigFile="myPrefs.js"
+Decoy="/tmp/imnotreal.js"
+AutoConfigScript="./firefox.cfg"
+AutoConfigLauncher="./autoconfig.js"
+
+PrefsFile=
+FFLibDir=
+
+initialize()
+{
+    kdb check prefs &>/dev/null
+
+    hasPrefsPlugin=$?
+
+    if [ "$hasPrefsPlugin" -ne 0 ]; 
+    then
+	echo "Error, prefs plugin not found"
+	exit 1
+    fi
+
+    kdb mount-info &>/dev/null
+
+    libDir=$(kdb get system/info/constants/package/libdir)
+    if [ -z "$libDir" ];
+    then
+	echo "Error, elektra libdir not found"
+	exit 1
+    fi
+
+    interceptFile="${libDir}/libelektraintercept.so"
+
+    if [ ! -e "$interceptFile" ];
+    then
+	echo "Error, libelektraintercept.so not found"
+	exit 1
+    fi
+
+    FFLibDir=$(find /usr/lib /usr/lib32 /usr/lib64 /usr/local/lib -name "firefox*" -type f | xargs dirname 2>/dev/null|sort|uniq)
+    if [ -z "$FFLibDir" ];
+    then
+	echo "Error, couldn't find any Firefox library directory"
+	exit 1
+    fi
+    resNo=$(echo "$FFLibDir" | wc -l)
+    if [ "$resNo" -gt 1 ];
+    then
+	i=1;
+	for line in ${FFLibDir};
+	do
+	    echo "$i) $line"
+	    i=$((++i))
+	done
+
+	echo -n "Select your Firefox library directory: "
+	read -r sel
+
+	FFLibDir=$(sed "${sel}q;d" <<< "$FFLibDir")
+    fi
+
+    echo "Firefox library directory: $FFLibDir"
+    echo
+
+    Profile=$(find "${HOME}/.mozilla/firefox/" -maxdepth 1 -mindepth 1 -type d 2>/dev/null)
+    if [ -z "$Profile" ];
+    then
+	echo "Couldn't finde Firefox profile directories in ${HOME}/.mozillla/Firefox/"
```

Comment:
some typos here ;) Please use a variable and do not repeat "${HOME}/.mozilla/firefox/